streaming/python/runtime/context_backend.py

```diff
@@ -0,0 +1,117 @@
+import logging
```

Comment:
name this file as `context.py` and put all context-related code here? Will this be better?

Reply:
I'll do this in separate PR. I don't want to change python code now, because it needs a lot of testing.